scalafmt-core/shared/src/main/scala/org/scalafmt/internal/BestFirstSearch.scala

```diff
@@ -151,7 +151,7 @@ class BestFirstSearch(
       stop: Token,
       depth: Int = 0,
       maxCost: Int = Integer.MAX_VALUE): State = {
-    val Q = new PriorityQueue[State]()
+    val Q = new mutable.PriorityQueue[State]()
```

Comment:
Is this change intentional? I think we have a custom `PriorityQueue` implementation that uses `java.util.PriorityQueue` instead of the one from the Scala collections, this was necessary for Graal native-image support and is not tested in the CI.